.storybook/preview.js

```diff
@@ -1,3 +1,34 @@
 import 'typeface-roboto'
 
-import './decorators.js'
+import React, { Fragment } from 'react'
+import { addDecorator } from '@storybook/react'
+
+import { CssReset } from '@dhis2/ui-core'
+
+addDecorator(fn => (
+    <Fragment>
+        <CssReset />
+        {fn()}
```

Comment:
hey @amcgee what is this `fn()` representing here? Are these stories that are being wrapped in this decorator?

Reply:
running late here.

Reply:
The `fn` function is a wrapper of the story itself and will merge all arguments passed to the `fn` with some other stuff and then pass it to the story (we're not passing anything to the function here though)

Reply:
Thanks @Mohammer5.

Reply:
That's right yeah (see [storybook docs](https://storybook.js.org/docs/basics/writing-stories/#decorators)) - this is identical to what was previously in `./decorators.js`, I just copied it here because Storybook appears to only allow the `addDecorator` in `preview.js`, not in imported modules (at least in built mode... not sure why, but this matches what we do in `ui-core` so aiming for consistency)

Reply:
That is clear. All I was getting to is that the name `fn` is a bit generic. But that is all fine since I was a little bit late after you finished reviewing and since there is consistency between this code base and the `ui-core` [here](https://github.com/dhis2/ui-core/blob/master/.storybook/preview.js#L12)

Reply:
@paschalidi yep agreed it’s a bit generic, but just keeping as-is here

Reply:
yep sure. :)